Userland/Libraries/LibIPC/Encoder.h

```diff
@@ -62,6 +64,13 @@ class Encoder {
         return *this;
     }
 
+    template<Enum T>
+    Encoder& encode(T const& enum_value)
+    {
+        *this << static_cast<std::underlying_type_t<T>>(enum_value);
```

Comment:
Same here, except you have to add an `#include <AK/StdLibExtras.h>`
```suggestion
        *this << static_cast<UnderlyingType<T>>(enum_value);
```

Reply:
`to_underlying(enum_value)` is a thing that exists : )